test/new/db/cmd/types

```diff
@@ -1476,7 +1476,7 @@ afs
 afs
 EOF
 EXPECT=<<EOF
-fcn.004e2420 (int64_t arg1, int64_t arg2, int64_t arg3, int64_t arg4, int64_t arg5, int64_t arg6, int64_t arg_8h);
```

Comment:
Is this. Orrect?

Reply:
by looking a bit at the assembly, this change seems wrong, yeah

Reply:
Yes, this is wrong, it's a bit weird because analyzing with `aaa` actually prints it with the arguments